backend/company_api.py

```
@@ -936,7 +936,154 @@ async def get_public_doctor_report():
+        )
+
+        return {"recommendations": recommendations}
+    except ImportError:
+        return {"recommendations": [], "message": "SkillBindings service not available"}
+
+
+@router.get("/{company_id}/specialists/{specialist_id}/skills")
+async def get_specialist_bound_skills(
+    company_id: str,
+    specialist_id: str,
+    user: dict = Depends(_get_current_user_thunk),
```

Ruff (0.15.15)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)